blueoil/blueoil_train.py

```diff
@@ -41,10 +41,17 @@ def save_config_file(config_file, dest_dir):
     if not gfile.Exists(dest_dir):
         gfile.MkDir(dest_dir)
 
+    config_file_dest = os.path.join(dest_dir, 'blueoil_config.yaml')
+
+    # HACK: This is for tensorflow bug workaround.
+    # We can remove lines 49-50 once it's been resolved in tensorflow
```

Comment:
It's better to simply say "following 2 lines" without referencing absolute line number like 49-50, because it'll slide if we add/remove lines above.